e2e-commons/docker-compose.yml

```diff
@@ -131,27 +129,27 @@ services:
       context: ..
       dockerfile: ui/Dockerfile
     environment:
-    - REACT_APP_HOME_BRIDGE_ADDRESS=0x32198D570fffC7033641F8A9094FFDCaAEF42624
-    - REACT_APP_FOREIGN_BRIDGE_ADDRESS=0x2B6871b9B02F73fa24F4864322CdC78604207769
-    - REACT_APP_FOREIGN_HTTP_PARITY_URL=http://localhost:8542
-    - REACT_APP_HOME_HTTP_PARITY_URL=http://localhost:8541
-    - REACT_APP_HOME_NATIVE_NAME=POA
-    - REACT_APP_HOME_NETWORK_NAME=Sokol
-    - REACT_APP_FOREIGN_NETWORK_NAME=Kovan
-    - REACT_APP_HOME_EXPLORER_TX_TEMPLATE=https://blockscout.com/poa/sokol/tx//%s
-    - REACT_APP_FOREIGN_EXPLORER_TX_TEMPLATE=https://blockscout.com/eth/kovan/tx/%s
-    - REACT_APP_HOME_EXPLORER_ADDRESS_TEMPLATE=https://blockscout.com/poa/sokol/address/%s
-    - REACT_APP_FOREIGN_EXPLORER_ADDRESS_TEMPLATE=https://blockscout.com/eth/kovan/address/%s
-    - REACT_APP_HOME_GAS_PRICE_ORACLE_URL=https://gasprice.poa.network/
-    - REACT_APP_HOME_GAS_PRICE_SPEED_TYPE=standard
-    - REACT_APP_HOME_GAS_PRICE_FALLBACK=5000000000
-    - REACT_APP_HOME_GAS_PRICE_UPDATE_INTERVAL=15000
-    - REACT_APP_HOME_GAS_PRICE_FACTOR=1
-    - REACT_APP_FOREIGN_GAS_PRICE_ORACLE_URL=https://gasprice.poa.network/
-    - REACT_APP_FOREIGN_GAS_PRICE_SPEED_TYPE=standard
-    - REACT_APP_FOREIGN_GAS_PRICE_FALLBACK=5000000000
-    - REACT_APP_FOREIGN_GAS_PRICE_UPDATE_INTERVAL=15000
-    - REACT_APP_FOREIGN_GAS_PRICE_FACTOR=1
+    - REACT_APP_COMMON_HOME_BRIDGE_ADDRESS=0x32198D570fffC7033641F8A9094FFDCaAEF42624
```

Comment:
Is it possible by somehow avoid usage of the `REACT_APP_` prefix here? For example - to refer to an `.env` file here instead of the variables and use pre-processing before running `docker-compose` to generate proper variables with correct prefix?

Reply:
Achieved in https://github.com/poanetwork/tokenbridge/pull/207